migrations/20221122004832_assessments__statistics__add.sql

```diff
@@ -0,0 +1,22 @@
+ALTER TABLE assessments ADD COLUMN statistics_last_updated_at timestamptz NOT NULL DEFAULT now() - interval '100 years';
+
+ALTER TABLE assessments ADD COLUMN score_stat_number integer NOT NULL DEFAULT 0;
+ALTER TABLE assessments ADD COLUMN score_stat_min double precision NOT NULL DEFAULT 0;
+ALTER TABLE assessments ADD COLUMN score_stat_max double precision NOT NULL DEFAULT 0;
+ALTER TABLE assessments ADD COLUMN score_stat_mean double precision NOT NULL DEFAULT 0;
+ALTER TABLE assessments ADD COLUMN score_stat_std double precision NOT NULL DEFAULT 0;
+ALTER TABLE assessments ADD COLUMN score_stat_median double precision NOT NULL DEFAULT 0;
+ALTER TABLE assessments ADD COLUMN score_stat_n_zero integer NOT NULL DEFAULT 0;
+ALTER TABLE assessments ADD COLUMN score_stat_n_hundred integer NOT NULL DEFAULT 0;
+ALTER TABLE assessments ADD COLUMN score_stat_n_zero_perc double precision NOT NULL DEFAULT 0;
+ALTER TABLE assessments ADD COLUMN score_stat_n_hundred_perc double precision NOT NULL DEFAULT 0;
+ALTER TABLE assessments ADD COLUMN score_stat_score_hist integer[] NOT NULL DEFAULT ARRAY[]::integer[];
```

Comment:
Nit: `score_stat_hist` might better match other `score_*` columns here (e.g. they aren't `store_stat_score_median`) and the duration below (`duration_stat_hist`, not `duration_stat_duration_hist`).

Reply:
Good call. `score_stat_score_hist` renamed in 1a72ee079f526c60e1950d168b4f8d516a074622. I believe `duration_stat_hist` is already called that (there is no `duration_stat_duration_hist `).

Reply:
Understood about the duration stats - was just pointing to that as an analogous column that was named how I wanted the score one to be!

Reply:
Ah, got it! Apologies for the confusion.